robots/lwr_sdh.urdf.xacro

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="lwr">
+
+  <xacro:arg name="use_gazebo" default="1"/>
+  <xacro:arg name="use_transmission" default="1"/>
+  <xacro:arg name="collision_simple" default="1"/>
+  <xacro:arg name="visual_simple" default="0"/>
+
+  <!-- KUKA LWR4 arm -->
+  <xacro:include filename="$(find kuka_lwr4_description)/robots/kuka_lwr_arm.urdf.xml"/>
+
+  <!-- Schunk SDH2 hand -->
+  <xacro:include filename="$(find schunk_description)/urdf/sdh/sdh.urdf.xacro"/>
```

Comment:
May requires a ``depend`` on ``schunk_description`` in the package.xml

Reply:
Right.